disco_aws_automation/disco_acm.py

```diff
@@ -6,27 +6,45 @@
 import boto3
 import botocore
 
+CERT_SUMMARY_LIST_KEY = 'CertificateSummaryList'
+CERT_ARN_KEY = 'CertificateArn'
+DOMAIN_NAME_KEY = 'DomainName'
+
 
 class DiscoACM(object):
     """
     A class to manage the Amazon Certificate Service
 
     """
+    WILDCARD_PREFIX = "*."
 
     def __init__(self, connection=None):
         self._acm = connection
 
+    def _in_domain(self, domain, dns_name):
```

Comment:
A *wildcard* cert of __\*.c.d__ will only work for __b.c.d__, not for __a.b.c.d__, which requires a *wildcard* cert like __\*.\*.c.d__